### PR TITLE
Add rails 5.1.1 to the test matrix

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

**Comment by jcoyne** [comment]
Looks like we're waiting on a new version of simple_form: https://github.com/plataformatec/simple_form/issues/1485

File: .travis.yml [hunk 1]
```diff
@@ -16,12 +16,13 @@ before_install:
...
   - "phantomjs --version"
 rvm:
-  - 2.4.0
+  - 2.4.1
 env:
   global:
     - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
   matrix:
     - "RAILS_VERSION=5.0.2"
+    - "RAILS_VERSION=5.1.0"
```
Comment (atz) [inline .travis.yml]: 👍

**Comment by jcoyne** [comment]
This is now blocked by #993

**Comment by mjgiarlo** [comment]
#993 is now merged.

**Pull request merged**